evaluation-tasks/util/luigi.py

```diff
@@ -37,14 +37,31 @@ def name(self):
         # return type(self).__name__
 
     def output(self):
-        return luigi.LocalTarget("_workdir/done-%s" % self.name)
+        return luigi.LocalTarget(
+            "_workdir/%02d-%s.done" % (self.stage_number, self.name)
```

Comment:
Nice one, really into this. This isn't specific to the PR, but how are we going to handle the _workdir when we start adding for pipelines? Should each pipeline get a separate dir inside of _workdir?

Reply:
_workdir/{config.TASKNAME}/